_partners/sematext.md

```diff
@@ -0,0 +1,81 @@
+---
+name: Sematext
+# name_long is used on the partner details page.
+name_long: 'Sematext Group'
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/sematext.png'
+link: 'https://sematext.com'
+# logo_large is used on the partner details page.
+logo_large: '/assets/media/partners/sematext.png'
+product_image: '/assets/media/partners/sematext.png'
+description: 'Sematext has deep expertise with OpenSearch and building search applications on top of it.  We've helped 100s of organizations with OpenSearch, Elasticsearch, and Solr.
```

Comment:
You probably want to double-quote this so that the all of the `We've`s don't prematurely end your description.

Reply:
Thanks.  Reworded to avoid the problem.